Gemfile

```diff
@@ -7,7 +7,7 @@ gem 'faraday-net_http_persistent', '~> 2.0'
 gem 'geo_combine'
 gem 'geoserver-publish', '~> 0.7.0'
 gem 'rsolr'
-gem 'rubyzip'
+gem 'rubyzip', '3.0.0.alpha'
```

Comment:
do we want to do pessimistic locking here (something like `gem 'rubyzip', '~> 3.0.0.alpha'`)? i *think* this will then retrieve `3.0.0` or higher if it's released...

Reply:
@anarchivist Thanks for catching that! I've updated it and pushed the changes to the branch.